### ISSUE TITLE
request for longer exposure duration

### ISSUE THREAD

**Issue by GoogleCodeExporter** [OpenPHDGuiding/phd2#424]
```
message from Terry:

I recently starting using a 14.5" iDK guided through a MMOAG with an SBIG ST-i. 
I have found guide stars very sparse when imaging galaxies toward the galactic 
pole!

My mount is well aligned and tracks very well. I have been using 15 seconds on 
PHD2 and the stars are very good. But I struggle for guide stars even at this 
long integration time.
I would like to try 30 seconds to see if I can better use faint guide stars 
with my ST-i guide cam. While I appreciate many would never consider such a 
long exposure, it could be very useful to a few of us using 'scopes with narrow 
fields of view and an OAG..

```

Original issue reported on code.google.com by `andy.gal...@gmail.com` on 17 Apr 2015 at 5:01

**Comment by agalasso** [comment]
Another user has requested longer exposure durations. In his case, the request is to support spectroscopy:

> I use long time exposures for see the object on the image and for moove it and put it in the slit of the spectro. 
> When it's ok and when i'm sure that the object are in the slit, i click on a star for start the guiding, and i change the time for beguing the guiding  with1,2, 3, secondes...

**Comment by d33psky** [comment]
How do we know if a camera is using the UI thread ?

**Comment by agalasso** [comment]
I just checked and *all* cameras in PHD2 currently use the worker thread (HasNonGuiCapture() returns true), i.e. no cameras use the main thread any more.

Unless we have reason to believe that a camera will be added in the future that requires capture on the main thread, we should remove HasNonGuiCapture() and the associated infrastructure for capturing on the main thread.

**Comment by agalasso** [comment]
fixed by 58ddb70702378fde667b635a8e6e8a4b287f7f0f
